stable/sealed-secrets/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: sealed-secrets
 description: A Helm chart for Sealed Secrets
-version: 1.11.0
-appVersion: 0.12.4
+version: 1.12.0
+appVersion: 0.13.1
```

Comment:
I think you need to at least update the tag in `values.yaml`. I will take a longer look at this later.

Reply:
Updated. Missed this file in first attempt :)